<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	color: white;	
	background-color: #264653;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="container">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="app_title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#e76f51; font-weight:700; font-size: 20pt; &quot;&gt;ĐỀ TÀI: CÁC THUẬT TOÁN MÃ HÓA CỔ ĐIỂN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="container_content">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>11</number>
          </property>
          <item>
           <widget class="QLabel" name="label_names">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777213</width>
              <height>39</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Tên thành viên nhóm:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_mk">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Lê Mai Khánh</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_nttn">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>Nguyễn Thị Tuyết Ngân</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_nqt">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>Ngô Quốc Tứ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_gvhd">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;GV hướng dẫn: TS. Phạm Đức Thành&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menu_mahoa">
     <property name="title">
      <string>Mã hóa</string>
     </property>
     <addaction name="action_mh_thaythe"/>
     <addaction name="action_mh_chuyenvi"/>
     <addaction name="action_mh_xor"/>
     <addaction name="action_mh_des"/>
    </widget>
    <widget class="QMenu" name="menu_giaima">
     <property name="title">
      <string>Giải mã</string>
     </property>
     <addaction name="action_gm_thaythe"/>
     <addaction name="action_gm_chuyenvi"/>
     <addaction name="action_gm_xor"/>
     <addaction name="action_gm_des"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_mahoa"/>
    <addaction name="separator"/>
    <addaction name="menu_giaima"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="action_mh_thaythe">
   <property name="text">
    <string>Thay thế</string>
   </property>
  </action>
  <action name="action_mh_chuyenvi">
   <property name="text">
    <string>Chuyển vị</string>
   </property>
  </action>
  <action name="action_mh_xor">
   <property name="text">
    <string>Xor</string>
   </property>
  </action>
  <action name="action_mh_des">
   <property name="text">
    <string>Des</string>
   </property>
  </action>
  <action name="action_gm_thaythe">
   <property name="text">
    <string>Thay thế</string>
   </property>
  </action>
  <action name="action_gm_chuyenvi">
   <property name="text">
    <string>Chuyển vị</string>
   </property>
  </action>
  <action name="action_gm_xor">
   <property name="text">
    <string>Xor</string>
   </property>
  </action>
  <action name="action_gm_des">
   <property name="text">
    <string>Des</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
